.editorconfig

```diff
@@ -0,0 +1,126 @@
+# http://EditorConfig.org
+# https://gds-way.cloudapps.digital/manuals/programming-languages/editorconfig
+# Copy into project root and rename to .editorconfig
+
+# uncomment if top-most EditorConfig file
+# root = true
+
+[*.java]
+indent_size = 4
+indent_style = space
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = false
+continuation_indent_size = 8
+
+[*.scss]
+indent_size = 2
+indent_style = space
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.erb]
+indent_size = 2
+indent_style = space
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Ah lot of this is the same - is there any way of setting it constant at the top or grouping the file types?

Reply:
This file comes direct from https://gds-way.cloudapps.digital/manuals/programming-languages.html and https://github.com/alphagov/gds-way/blob/main/source/manuals/programming-languages/editorconfig - so any changes would need to be made upstream

Reply:
Fair - no worries then